AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_Horovod_Multinode_Training/TensorFlow_Multinode_Training_with_Horovod.py

```diff
@@ -44,7 +44,7 @@
 tf.compat.v1.disable_eager_execution()
 '''
 Environment settings:
-Set MKLDNN_VERBOSE=1 to show DNNL run time verbose
+Set MKLDNN_VERBOSE=1 to show Intel® Deep Neural Network Library (Intel® DNNL) run time verbose
```

Comment:
Same question re rich text in .py files?

Reply:
removed this until I have a definitive answer